pkg/distro/defs/loader.go

```diff
@@ -68,6 +68,7 @@ type DistroYAML struct {
 	// for {{.Major}}, {{.Minor}} which will be expanded by the
 	// upper layers.
 	Name             string            `yaml:"name"`
+	Aliases          []string          `yaml:"name_aliases"`
```

Comment:
As far as I can tell, this is not necessary, because it adds `name_aliases` for the distribution, not the image type.

At this point, we should not be handling aliases for distributions in `images`, since this also affects repo config loading. Moreover, distro aliases are, for now, intentionally handled on the higher levels - osbuild-composer. So if we were to bring that feature to `images`, so that, i.e., ibcli can use it, we should bring it from osbuild-composer, together with the unit and functional tests that we have for it.